docs/Displaying a User Interface/Text Images and Buttons/index.md

```diff
@@ -59,10 +60,28 @@ sdlManager.screenManager.endUpdates { (error) in
     }
 }
 ```
+!@
+
+@![android, javaSE, javaEE]
+```java
+sdlManager.getScreenManager().beginTransaction();
+sdlManager.getScreenManager().setTextField1("Hello, this is MainField1.");
+sdlManager.getScreenManager().setTextField2("Hello, this is MainField2.");
+sdlManager.getScreenManager().setTextField3("Hello, this is MainField3.");
+sdlManager.getScreenManager().setTextField4("Hello, this is MainField4.");
+sdlManager.getScreenManager().commit(new CompletionListener() {
+	@Override
+	public void onComplete(boolean success) {
+		Log.i(TAG, "ScreenManager update complete: " + success);
+	}
+});
+```
+!@
 
 ### Removing Text and Images
 After you have displayed text and graphics onto the screen, you may want to remove those from being displayed. In order to do so, you only need to set the screen manager property to `nil`.
```

Comment:
Should this be "set the screen manager property to @![iOS]`nil`!@ @![android,javaSE,javaEE]`null`" so the description can match the code example for both iOS and Android

Reply:
Fixed.